packages/backend/src/handlers/remote-schemas/typeDefs.ts

```diff
@@ -1,9 +1,19 @@
 import { gql } from 'graphql-request/dist';
 
 export const typeDefs = gql`
+  scalar uuid
+
   type Query {
     getBoxProfile(address: String): BoxProfile
     getDaoHausMemberships(memberAddress: String): [Member!]!
+    getBrightIdStatus(contextId: uuid): BrightIdStatus
```

Comment:
Without this Hasura was giving me an error when creating a remote relationship.
See https://github.com/hasura/graphql-engine/issues/5482